Vesper-Agents-Suite/Fund/src/utils.ts

```diff
@@ -1,8 +1,9 @@
 import { Finding, FindingSeverity, FindingType } from "forta-agent";
-import Web3 from "web3";
-import { ControllerABI, AddressListABI } from "./abi";
-
-const controllerAddress = "0xa4F1671d3Aee73C05b552d57f2d16d3cfcBd0217";
+const axios = require("axios");
+export const API_URL = "https://api.vesper.finance/dashboard?version=2";
+const STATUS = "operative";
+const ALLOWED_STAGES = ["prod", "beta", "orbit"];
+const IGNORE_ADDRESS = ["0xbA4cFE5741b357FA371b506e5db0774aBFeCf8Fc"];
```

Comment:
Could you add a comment to say what the IGNORE_ADDRESS is.  I know it is vVSP pool, but I think it would help to have it written down here if someone other than us is reading it (or if we read it later one).